paging.json

```diff
@@ -21,6 +21,7 @@
         "x-ms-pageable": {
           "nextLinkName": "@odata.nextLink",
           "itemName": "values",
+          "operationName": "examplePagingNext"
```

Comment:
you need to add a new operation definition in the swagger, and link to it through field `operationName` in [`x-ms-pageable`](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-pageable)